src/backend/utils/adt/age_global_graph.c

```diff
@@ -41,7 +41,7 @@
 
 /* internal data structures implementation */
 
-/* vertex entry for the vertex_hastable */
+/* vertex entry for the vertex_hashtable */
```

Comment:
?

Reply:
This one need to be checked in source code as well.